import React, { Component } from 'react';
import styles from '../AppsView.scss';

import Pipeline from 'components/Pipeline';

export default class DockerizedPipelineModule extends Component {
  constructor(props) {
    super(props);
    this.state = {
      sortBy: 'Shuffled',
    };
  }

  render() {
    return (

      <div className="row">
        <h3
          style={{ paddingLeft: '0.9375rem' }}
          className={styles['section-title']}
        >
          Dockerized Pipelines
        </h3>
        <br />
      {
        <div className="col-xs-12 col-md-12 col-xl-6">
          <Pipeline
            name="MEMA"
            title="Docker container for using MEMA R package in RStudio"
            description="This docker container provides a platform for
            processing and the analysis of microenvironment microarray (mema)
            data generated by The Microenvironment Perturbagen LINCS (MEP LINCS)
            Center at the Oregon Health and Science University based
            platform assesses. The platform is provided in the form of an
            RStudio instance. The mema docker container was built based on the
            validated MEMA R package v1.0.1 (released on 2017-05-16) to run all
            the R code in the vignettes created by MEP-LINCS
            project on 2017-05-17
            (https://github.com/MEP-LINCS/MEMA/tree/master/vignettes)."
            githubUrl="https://github.com/uc-bd2k/mema"
            dockerHubUrl="https://hub.docker.com/r/ucbd2k/mema/"
            toolTipItems="The processing pipeline implemented in the vignettes
            starts with Level 1 data and produces Level 2, Level 3 and Level 4
            along with interactive visualization of data."
          />
        </div>
      }
      {
        <div className="col-xs-12 col-md-12 col-xl-6">
          <Pipeline
            name="PCCSE"
            title="Docker container for running PCCSE processing pipelines in RStudio"
            description="This docker container provides a platform for the
            processing of P100 and GCP data generated by LINCS Proteomic
            Characterization Center for Signaling and Epigenetics (PCCSE) and
            generate level 4 data from level 2 data. The platform is provided
            in the form of an RStudio instance. The docker is tested on
            August-31-2017 on the level 2 of P100 and GCP data from panorama."
            githubUrl="https://github.com/uc-bd2k/pccse"
            dockerHubUrl="https://hub.docker.com/r/ucbd2k/pccse/"
            toolTipItems="The processing pipeline starts with Level 2 data (
            raw numerical data, RPT) and ultimately produces Level 4 data
            (differential quantification, DIFF)."
          />
        </div>
      }
      {
        <div className="col-xs-12 col-md-12 col-xl-6">
          <Pipeline
            name="NL_RNASEQ"
            title="Docker container for generating NeuroLINCS RNA-seq signature (Step-2) in RStudio"
            description="This docker container provides a platform for
            processing and the analysis of next generation sequencing (RNA-seq)
            data generated by NeuroLINCS collaborative center between University
            of California- Irvine, Cedars Sinai, the Gladstone Institute, MIT,
            and Johns Hopkins University."
            githubUrl="https://github.com/uc-bd2k/Nl_rnaseq"
            dockerHubUrl="https://hub.docker.com/r/ucbd2k/nl_rnaseq/"
            toolTipItems="The platform is provided in the form of an RStudio
            instance.The analysis scripts within the the container create
            NeuroLINCS RNA-seq level 4 signature data (differential expression
            profile) from level 3 (raw counts) data."
          />
        </div>
      }
      {
        <div className="col-xs-12 col-md-12 col-xl-6">
          <Pipeline
            name="HMS-CELLCOUNT"
            title="Docker container for generating Growth-rate inhibition (GR)
            values and fitted curve metrics in RStudio"
            description="This docker container provides a platform for
            processing and the analysis of cancer cell line drug-response assays
            generated by the HMS LINCS Data and Signatures Generation Center.
            The platform is provided in the form of an RStudio instance."
            githubUrl="https://github.com/uc-bd2k/hms-cellcount/"
            dockerHubUrl="https://hub.docker.com/r/ucbd2k/hms-cellcount/"
            toolTipItems="The processing pipeline implemented in a form of an R
            script starts with Level 2 data (cell counts) and create Level 3
            data (average cell counts) and Level 4 data (dose response curve
            metrics). The scripts average cell count data over technical
            replicates (estimated by intra-cellular ATP levels measured by a
            CellTiter-GloÂ® assay) and fit the data to 3-parameter logistic
            dose-response curves using the growth-rate inhibition (GR)
            method."
          />
        </div>
      }
      </div>
    );
  }
}
