module.exports = [{"icon_url": "http://amp.pharm.mssm.edu/LINCS/files/enrichr.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "Enrichr is an easy to use intuitive enrichment analysis web-based tool providing various types of visualization summaries of collective functions of gene lists.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": true, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://amp.pharm.mssm.edu/Enrichr", "clust_l1000_data": true, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": true, "api": false, "feature_browse": false, "name": "Enrichr", "web_based_ui": true}, {"icon_url": "http://amp.pharm.mssm.edu/LINCS/files/g2e.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "GEO2Enrichr is a Google Chrome Extension and Python server developed to ease the extraction of signatures from studies posted on the Gene Expression Omnibus (GEO) database.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": true, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://amp.pharm.mssm.edu/g2e", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": true, "api": false, "feature_browse": false, "name": "GEO2Enrichr", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/docent-grid.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "Docent III provides an interactive matrix of the most studied cell lines by assay.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://amp.pharm.mssm.edu/milestones/grid.html", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "Docent III", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/portal.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "The current version of the portal has features for searching and exploring LINCS dataset packages and reagents", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": false, "url": "http://lincsportal.ccs.miami.edu/dcic-portal/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": true, "feature_browse": true, "name": "LINCS Data Portal", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/pilincs.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "Provides interface and seamless access to LINCS proteomics datasets (P100, GCP, etc.) available in\u00cc\u00c4\u00e5_Panorama. Intermediate API layer to enable direct and simple querying and access to proteomics datasets by tools developed within BD2K community.\u00cc\u00c4\u00e5_Facilitates data integration by creating a global view of L1000 and proteomic data through\u00cc\u00c4\u00e5_iLINCS\u00cc\u00c4\u00e5_and related portals.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://eh3.uc.edu/pilincs/#/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": true, "feature_browse": false, "name": "piLINCS", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/slicr.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "Slicr is a metadata search engine that searches for LINCS L1000 gene expression profiles and signatures (for GSE70138 only) matching user\u00cc\u00c4\u00e5\u008ds input parameters.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": true, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://amp.pharm.mssm.edu/Slicr", "clust_l1000_data": true, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": true, "api": false, "feature_browse": false, "name": "Slicr", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/l1000cds2.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "L1000CDS2 queries gene expression signatures against the LINCS L1000 to identify and prioritize small molecules that can reverse or mimic the observed input expression pattern.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": true, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://amp.pharm.mssm.edu/L1000CDS2", "clust_l1000_data": true, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": true, "api": false, "feature_browse": false, "name": "L1000CDS2 - L1000 Characteristic Direction Signature Search Engine", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/paea.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "PAEA is a new R/Shiny gene set enrichment web application with over 70 gene set libraries available for enrichment analysis.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": true, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://amp.pharm.mssm.edu/PAEA", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "Principal Angle Enrichment Analysis (PAEA)", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/life.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "A semantically-enhanced web-based application that enables access, navigation and exploration of a knowledgebase built by integrating and indexing all the LINCS data types. LIFE allows access, navigation and exploration of LINCS assays, biomolecules, related concepts and LINCS screening results via a variety of views such as proteins, genes, cell lines, small molecules. LIFE provides flexible navigation of the LINCS assay and data landscape via list functionality covering important assay biomolecules and concepts; this enables a variety of use cases.", "feature_query": true, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": true, "feature_download": true, "feature_vizualization": false, "url": "http://life.ccs.miami.edu/life/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": true, "feature_browse": true, "name": "LIFE", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/i-lincs.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "iLINCS is a portal that handles LINCS L1000 and KinomeScan data. It facilitates integration of LINCS data-derived signatures with other genome-scale signatures.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": true, "feature_download": true, "feature_vizualization": true, "url": "http://eh3.uc.edu/GenomicsPortals/Lincs.jsp", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": false, "feature_leverage_ontology": true, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": true, "feature_browse": false, "name": "iLINCS", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/lincs-canvas-browser.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "Compact visualization of thousands of L1000 experiments; clustering of perturbations based on signature similarity; interactive gene list enrichment analysis using 32 gene set libraries; query up and down gene lists against over 140,000 L1000 conditions.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": true, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://www.maayanlab.net/LINCS/LCB/", "clust_l1000_data": true, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": true, "api": false, "feature_browse": false, "name": "LINCS Canvas Browser", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/drug-cell-line-browser.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "Drug/Cell-line Browser (DCB), an online interactive HTML5 data visualization tool for interacting with three of the recently published datasets of cancer cell lines/drug-viability studies. DCB uses clustering and canvas visualization of the drugs and the cell lines, as well as a bar graph that summarizes drug effectiveness for the tissue of origin or the cancer subtypes for single or multiple drugs. DCB can help in understanding drug response patterns and prioritizing drug/cancer cell line interactions by tissue of origin or cancer subtype.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": true, "feature_navigation": false, "feature_download": false, "feature_vizualization": true, "url": "http://www.maayanlab.net/LINCS/DCB/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "Drug/Cell-line Browser", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/network2canvas.png", "center": "BD2K-LINCS DCIC", "client_server_software": false, "description": "N2C provides a new flexible method to visualize large networks and can be used to perform and visualize gene-set and drug-set enrichment analyses.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": true, "feature_mining": false, "feature_navigation": true, "feature_download": false, "feature_vizualization": true, "url": "http://www.maayanlab.net/N2C/", "clust_l1000_data": true, "feature_image_analysis": false, "feature_access": true, "feature_search": false, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": true, "api": false, "feature_browse": false, "name": "Network2Canvas", "web_based_ui": true}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/hms_lincs.png", "center": "HMS LINCS", "client_server_software": false, "description": "The HMS LINCS Database contains documentation for HMS LINCS reagents (small molecule and protein perturbagens, cells, and other protein reagents). Multiple HMS LINCS datasets are also available in this database for browsing and download; for several datasets, primary images are available to view and download. The HMS LINCS Database evolves continuously and new features and data will be available on an ongoing basis. It will not be complete per se, since we intend to continue to modify it throughout the project.", "feature_query": true, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": true, "feature_download": true, "feature_vizualization": false, "url": "https://lincs.hms.harvard.edu/db/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": true, "feature_browse": true, "name": "HMS LINCS Database", "web_based_ui": true}, {"icon_url": "http://dev.lincs.hms.harvard.edu/wordpress/wp-content/uploads/2011/06/ImageRail_image_crop.jpg", "center": "HMS LINCS", "client_server_software": false, "description": "Desktop software for image analysis of high-throughput microscopy imagery. It stores single-cell data using SDCubes and offers an adaptive experimental designer. ImageRail is written in Java and is thus cross-platform, and native application installers are provided for Windows and Mac OS X. ImageRail is described in detail in\u00cc\u00c4\u00e5_Millard et al. 2011. ImageRail is used to collect and analyze data from many of the HMS LINCS fixed-cell microscopy assays.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": true, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": false, "feature_download": false, "feature_vizualization": false, "url": "https://github.com/sorgerlab/ImageRail", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": true, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "ImageRail", "web_based_ui": false}, {"icon_url": "http://www.openmicroscopy.org/site/products/images/omero191x120.png", "center": "HMS LINCS", "client_server_software": true, "description": "Client-server software for visualization, management and analysis of biological microscope images.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": true, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": false, "feature_download": false, "feature_vizualization": true, "url": "http://www.openmicroscopy.org/site", "clust_l1000_data": false, "feature_image_analysis": true, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "OMERO", "web_based_ui": false}, {"icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore2.jpg", "center": "HMS LINCS", "client_server_software": false, "description": "Interactive map of key signal transduction pathways and relevant drug-target data.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": false, "feature_download": false, "feature_vizualization": true, "url": "http://lincs.hms.harvard.edu/explore/pathway/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "Kinase Inhibitor Pathway Explorer", "web_based_ui": true}, {"icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore4.jpg", "center": "HMS LINCS", "client_server_software": false, "description": "Interactive exploration tool to investigate the responses of 50+ cell lines to 64 anti-cancer drugs.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": true, "feature_navigation": false, "feature_download": false, "feature_vizualization": true, "url": "http://lincs.hms.harvard.edu/explore/10.1038-nchembio.1337/fallahi-sichani-2013/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "Interactive Drug Response Data Browser", "web_based_ui": true}, {"icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore-breast-cancer-signaling.jpg", "center": "HMS LINCS", "client_server_software": false, "description": "Comprehensive analysis matrix showing ERK and Akt responses to growth factor stimulation.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": true, "feature_navigation": false, "feature_download": false, "feature_vizualization": true, "url": "http://lincs.hms.harvard.edu/explore/breast-cancer-signaling/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "Breast Cancer Cell Line Signaling Data Matrix", "web_based_ui": true}, {"icon_url": "http://lincs.hms.harvard.edu/_static/single_cell_dynamics/g/img/cell/r4c8.png", "center": "HMS LINCS", "client_server_software": false, "description": "Live-cell data visualization describing the responses of cells to 3 EGFR/Her2 inhibitors.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": true, "feature_navigation": false, "feature_download": false, "feature_vizualization": true, "url": "http://lincs.hms.harvard.edu/explore/single-cell-dynamics/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "Dynamics of Perturbagen Response in Single Living Cells", "web_based_ui": true}, {"icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore-tiedie.jpg", "center": "HMS LINCS", "client_server_software": false, "description": "The TieDIE algorithm computes a subnetwork of gene and protein interactions that connect genomic perturbations to gene expression changes characteristic of cancer subtypes. The pathways derived in this way predict interlinking genes that may correspond to essential components of cancer signaling and may provide a mechanistic explanation of tumor behavior and suggest subtype-specific drug targets. In addition, the algorithm can extend to generate connecting subnetworks for generic gene sets, and runs a null-model permutation test to determine if these sets are significantly close in pathway space.", "feature_query": false, "matlab_python_script": true, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": true, "feature_navigation": false, "feature_download": false, "feature_vizualization": true, "url": "https://sysbiowiki.soe.ucsc.edu/tiedie", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "TieDIE", "web_based_ui": false}, {"icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/cmap2.jpg", "center": "Broad Transcriptomics", "client_server_software": false, "description": "Web apps developed by the Broad Institute allow users to query the over 1.4 million gene expression experiments in the LINCS/CMap database hosted on lincscloud.org without having to download the data.", "feature_query": true, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": true, "feature_download": true, "feature_vizualization": false, "url": "http://apps.lincscloud.org/", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": true, "feature_search": true, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": true, "feature_browse": true, "name": "Lincscloud Apps", "web_based_ui": true}, {"icon_url": "http://amp.pharm.mssm.edu/LINCS/files/mep-lincs-data-explorer.png", "center": "MEP LINCS", "client_server_software": false, "description": "Interactive visualization of the microenvironment microarray data collected at MEP LINCS. This R/Shiny tool provides access to different visualization endpoints for human cancer cell lines placed in different microenvironments and treated with an array of endogenous ligands.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": false, "feature_download": false, "feature_vizualization": false, "url": "https://www.synapse.org/#!Synapse:syn2862345/wiki/231720", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": false, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "LINCS Data Explorer", "web_based_ui": true}, {"icon_url": "http://www.neurolincs.org/assets/lincs-bg.png", "center": "NeuroLINCS", "client_server_software": false, "description": "A data integration tool for transcriptomic and epigenomic data. This tool generates a list of enriched motifs in open chromatin regions (as assayed by ATAC-seq or DNAseH) for a given set genes. The foreground list of genes are differentially expressed or highly expressed genes and the background list of genes are the rest of the genes in the transcriptomic experiment.", "feature_query": false, "matlab_python_script": false, "feature_image_manage": false, "desktop_software": false, "enrichment_analysis": false, "feature_mining": false, "feature_navigation": false, "feature_download": false, "feature_vizualization": false, "url": "http://www.neurolincs.org/resources/AChroMap", "clust_l1000_data": false, "feature_image_analysis": false, "feature_access": false, "feature_search": false, "feature_integration": true, "feature_leverage_ontology": false, "feature_data_analysis": false, "feature_signature_enrichment": false, "api": false, "feature_browse": false, "name": "AChroMap", "web_based_ui": true}]