module.exports = [{"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": true, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://amp.pharm.mssm.edu/Enrichr", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://amp.pharm.mssm.edu/LINCS/files/enrichr.png", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": true, "name": "Enrichr", "feature_visualization": true, "description": "Enrichr is an easy to use intuitive enrichment analysis web-based tool providing various types of visualization summaries of collective functions of gene lists.", "order": 13}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://amp.pharm.mssm.edu/g2e", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://amp.pharm.mssm.edu/LINCS/files/g2e.png", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": false, "name": "GEO2Enrichr", "feature_visualization": true, "description": "GEO2Enrichr is a Google Chrome Extension and Python server developed to ease the extraction of signatures from studies posted on the Gene Expression Omnibus (GEO) database.", "order": 14}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": true, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://lincsportal.ccs.miami.edu/dcic-portal/", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/portal.png", "feature_access": true, "enrichment_analysis": false, "api": true, "feature_image_analysis": true, "processed_l1000_data": true, "name": "LINCS Data Portal", "feature_visualization": true, "description": "The LINCS Data Portal provides a unified interface for searching LINCS dataset packages and reagents", "order": 1}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://eh3.uc.edu/pilincs/#/", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/pilincs.png", "feature_access": true, "enrichment_analysis": false, "api": true, "feature_image_analysis": false, "processed_l1000_data": false, "name": "piLINCS", "feature_visualization": true, "description": "Provides interface and seamless access to LINCS proteomics datasets (P100, GCP, etc.) available in Panorama.", "order": 4}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://amp.pharm.mssm.edu/Slicr", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/slicr.png", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": true, "name": "Slicr", "feature_visualization": true, "description": "Slicr is a metadata search engine that searches for LINCS L1000 gene expression profiles and signatures matching user's input parameters. It features download of selected search results as csv files in a zipped folder and visualization of selected results in a 3D scatter plot using PCA or MDS. Slicr consists of three views: the search view, the checkout view and the 3D scatter view.", "order": 5}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://amp.pharm.mssm.edu/L1000CDS2", "feature_navigation": true, "feature_search": true, "feature_download": false, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/l1000cds2.png", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": true, "name": "L1000CDS2 - L1000 Characteristic Direction Signature Search Engine", "feature_visualization": true, "description": "1000CDS2 queries gene expression signatures against the LINCS L1000 to identify and prioritize small molecules that can reverse or mimic the observed input expression pattern.", "order": 3}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": true, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://amp.pharm.mssm.edu/PAEA", "feature_navigation": true, "feature_search": true, "feature_download": false, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/paea.png", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": false, "name": "PAEA - Principal Angle Enrichment Analysis", "feature_visualization": true, "description": "Functional analysis of genome-wide differential expression is central to biological investigations. Here we present a new multivariate approach to gene-set enrichment called Principal Angle Enrichment Analysis (PAEA). PAEA uses the geometrical concept of the principal angle to quantify gene-set enrichment. We find that PAEA outperforms a selection of commonly used gene set enrichment methods including GSEA. To benchmark PAEA with other enrichment methods we use real data. We examined the ranking of transcription factors by performing enrichment analysis on gene expression signatures from many studies that knocked-down, knocked-out or over-expressed transcription factors, and performed the enrichment analysis with a library of gene sets created from ChIP-Seq data profiling the same transcription factors. We also found that PAEA was able to rank better aging-related phenotype-terms from a collection of gene expression profiling studies where tissue from young adults was compared to tissue of elderly subjects. PAEA is implemented as a user-friendly R Shiny gene-set enrichment web application with over 70 gene set libraries available for enrichment analysis. Canned enrichment analysis for over 700 disease signatures extracted from GEO is provided with the application which is freely available at: http://amp.pharm.mssm.edu/PAEA .", "order": 15}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": true, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://eh3.uc.edu/GenomicsPortals/Lincs.jsp", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/i-lincs.png", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": true, "name": "iLINCS", "feature_visualization": true, "description": "iLINCS (Integrative LINCS) is an integrative web platform for analysis of LINCS data and signatures. The portal provides biologists-friendly user interfaces for analyzing transcriptomics and proteomics LINCS datasets. The portal integrates R analytical engine via several R tools for web-computing (rserve, opencpu, Shiny, rgl) and DCIC developed web tools and applications (FTreeView, Enrichr) into a coherent web platform for LINCS data analysis. Users can follow several workflows which allow them to identify differentially expressed genes, proteins and phosphoproteins in LINCS datasets and use them in analysis of other LINCS and non-LINCS dataset (eg TCGA and GEO transcriptomic datasets), and in the analysis of LINCS L1000 signatures. In this way, the platform facilitates integrative analysis of LINCS data and signatures.", "order": 2}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://www.maayanlab.net/LINCS/LCB/", "feature_navigation": true, "feature_search": true, "feature_download": false, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/lincs-canvas-browser.png", "feature_access": true, "enrichment_analysis": true, "api": false, "feature_image_analysis": false, "processed_l1000_data": true, "name": "LINCS Canvas Browser", "feature_visualization": true, "description": "Compact visualization of thousands of L1000 experiments; clustering of perturbations based on signature similarity; interactive gene list enrichment analysis using 32 gene set libraries; query up and down gene lists against over 140,000 L1000 conditions.", "order": 10}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://www.maayanlab.net/LINCS/DCB/", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/drug-cell-line-browser.png", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": false, "processed_l1000_data": false, "name": "Drug/Cell-line Browser", "feature_visualization": true, "description": "Drug/Cell-line Browser (DCB), an online interactive HTML5 data visualization tool for interacting with three of the recently published datasets of cancer cell lines/drug-viability studies. DCB uses clustering and canvas visualization of the drugs and the cell lines, as well as a bar graph that summarizes drug effectiveness for the tissue of origin or the cancer subtypes for single or multiple drugs. DCB can help in understanding drug response patterns and prioritizing drug/cancer cell line interactions by tissue of origin or cancer subtype.", "order": 11}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": true, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://amp.pharm.mssm.edu/Harmonizome/", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://amp.pharm.mssm.edu/Harmonizome/image/logo_100x87.png", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": true, "name": "Harmonizome", "feature_visualization": true, "description": "Thanks to technological advances in genomics, transcriptomics, proteomics, metabolomics, and related fields, projects that generate a large number of measurements of the properties of cells, tissues, model organisms, and patients are becoming commonplace in biomedical research. In addition, curation projects are making great progress mining biomedical literature to extract and aggregate decades worth of research findings into online databases. Such projects are generating a wealth of information that potentially can guide research toward novel biomedical discoveries and advances in healthcare. To facilitate access to and learning from biomedical Big Data, we created the Harmonizome: a collection of information about genes and proteins from 114 datasets provided by 66 online resources.", "order": 12}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "BD2K-LINCS DCIC", "url": "http://www.phosphosite.org/", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://www.phosphosite.org/images/top_banner.gif", "feature_access": true, "enrichment_analysis": false, "api": true, "feature_image_analysis": false, "processed_l1000_data": false, "name": "Phosphosite Plus", "feature_visualization": false, "description": "Online resource providing comprehensive information and tools for the study of protein post-translational modifications (PTMs)", "order": 23}, {"feature_integration": false, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": false, "center": "BD2K-LINCS DCIC", "url": "https://github.com/WebDataScience/GUIdock", "feature_navigation": false, "feature_search": false, "feature_download": false, "icon_url": "https://raw.githubusercontent.com/kristiyanto/GUIdock/master/logo.png", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": false, "processed_l1000_data": false, "name": "GUIDock", "feature_visualization": false, "description": "Using Docker containers with a common graphics user interface to address the reproducibility of research", "order": 24}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": true, "web_based_ui": true, "center": "HMS LINCS", "url": "https://lincs.hms.harvard.edu/db/", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/hms_lincs.png", "feature_access": true, "enrichment_analysis": false, "api": true, "feature_image_analysis": true, "processed_l1000_data": true, "name": "HMS LINCS Database", "feature_visualization": true, "description": "The HMS LINCS Database contains documentation for HMS LINCS reagents (small molecule and protein perturbagens, cells, and other protein reagents). Multiple HMS LINCS datasets are also available in this database for browsing and download; for several datasets, primary images are available to view and download. The HMS LINCS Database evolves continuously and new features and data will be available on an ongoing basis. It will not be complete per se, since we intend to continue to modify it throughout the project.", "order": 7}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "HMS LINCS", "url": "http://lincs.hms.harvard.edu/explore/pathway/", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore2.jpg", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": false, "processed_l1000_data": false, "name": "Kinase Inhibitor Pathway Explorer", "feature_visualization": true, "description": "Interactive map of key signal transduction pathways and relevant drug-target data.", "order": 17}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "HMS LINCS", "url": "http://lincs.hms.harvard.edu/explore/10.1038-nchembio.1337/fallahi-sichani-2013/", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore4.jpg", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": true, "processed_l1000_data": false, "name": "Interactive Drug Response Data Browser", "feature_visualization": true, "description": "Interactive exploration tool to investigate the responses of 50+ cell lines to 64 anti-cancer drugs.", "order": 18}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "HMS LINCS", "url": "http://lincs.hms.harvard.edu/explore/breast-cancer-signaling/", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore-breast-cancer-signaling.jpg", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": true, "processed_l1000_data": false, "name": "Breast Cancer Cell Line Signaling Data Matrix", "feature_visualization": true, "description": "Comprehensive analysis matrix showing ERK and Akt responses to growth factor stimulation.", "order": 19}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "HMS LINCS", "url": "http://lincs.hms.harvard.edu/explore/single-cell-dynamics/", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://lincs.hms.harvard.edu/_static/single_cell_dynamics/g/img/cell/r4c8.png", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": true, "processed_l1000_data": false, "name": "Dynamics of Perturbagen Response in Single Living Cells", "feature_visualization": true, "description": "Live-cell data visualization describing the responses of cells to 3 EGFR/Her2 inhibitors.", "order": 20}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "HMS LINCS", "url": "http://lincs.hms.harvard.edu/explore/adaptive-drug-resistance/", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://lincs.hms.harvard.edu/_static/adaptive_drug_resistance/img/vips/thumbnail.png", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": true, "processed_l1000_data": false, "name": "Adaptive Drug Resistance Browser", "feature_visualization": true, "description": "Systematic analysis of BRAFV600E melanomas reveals a role for JNK/c-Jun pathway in adaptive resistance to drug-induced apoptosis", "order": 21}, {"feature_integration": false, "matlab_python_script": true, "feature_leverage_ontology": false, "web_based_ui": false, "center": "HMS LINCS", "url": "https://sysbiowiki.soe.ucsc.edu/tiedie", "feature_navigation": false, "feature_search": false, "feature_download": false, "icon_url": "http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2012/10/explore-tiedie.jpg", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": false, "processed_l1000_data": false, "name": "TieDIE", "feature_visualization": true, "description": "The TieDIE algorithm computes a subnetwork of gene and protein interactions that connect genomic perturbations to gene expression changes characteristic of cancer subtypes. The pathways derived in this way predict interlinking genes that may correspond to essential components of cancer signaling and may provide a mechanistic explanation of tumor behavior and suggest subtype-specific drug targets. In addition, the algorithm can extend to generate connecting subnetworks for generic gene sets, and runs a null-model permutation test to determine if these sets are significantly close in pathway space.", "order": 16}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "Broad Transcriptomics", "url": "http://apps.lincscloud.org/", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://lincs-dcic.org/libs/lincs-direct-access-tools/tool-icons/cmap2.jpg", "feature_access": true, "enrichment_analysis": true, "api": true, "feature_image_analysis": false, "processed_l1000_data": true, "name": "Lincscloud Apps", "feature_visualization": true, "description": "Web apps developed by the Broad Institute allow users to query the over 1.4 million gene expression experiments in the LINCS/CMap database hosted on lincscloud.org without having to download the data.", "order": 6}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "MEP LINCS", "url": "https://www.synapse.org/#!Synapse:syn2862345/wiki/231720", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://amp.pharm.mssm.edu/LINCS/files/mep-lincs-data-explorer.png", "feature_access": true, "enrichment_analysis": false, "api": false, "feature_image_analysis": true, "processed_l1000_data": false, "name": "LINCS Data Explorer", "feature_visualization": true, "description": "Interactive visualization of the microenvironment microarray data collected at MEP LINCS. This R/Shiny tool provides access to different visualization endpoints for human cancer cell lines placed in different microenvironments and treated with an array of endogenous ligands.", "order": 22}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "NeuroLINCS", "url": "http://www.neurolincs.org/resources/AChroMap", "feature_navigation": true, "feature_search": false, "feature_download": false, "icon_url": "http://www.neurolincs.org/assets/lincs-bg.png", "feature_access": false, "enrichment_analysis": false, "api": false, "feature_image_analysis": false, "processed_l1000_data": false, "name": "AChroMap", "feature_visualization": true, "description": "A data integration tool for transcriptomic and epigenomic data. This tool generates a list of enriched motifs in open chromatin regions (as assayed by ATAC-seq or DNAseH) for a given set genes. The foreground list of genes are differentially expressed or highly expressed genes and the background list of genes are the rest of the genes in the transcriptomic experiment.", "order": 8}, {"feature_integration": true, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "DToxS", "url": "https://martip03.u.hpc.mssm.edu", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "http://amp.pharm.mssm.edu/LINCS/files/dtoxs-screenshot.png", "feature_access": true, "enrichment_analysis": false, "api": false, "feature_image_analysis": true, "processed_l1000_data": false, "name": "DToxS Website", "feature_visualization": true, "description": "Web portal to provide access to the DToxS LINCS center data, metadata and analyses", "order": 9}, {"feature_integration": false, "matlab_python_script": false, "feature_leverage_ontology": false, "web_based_ui": true, "center": "LINCS PCCSE", "url": "https://panoramaweb.org/labkey/wiki/LINCS/Overview%20Information/page.view?name=LINCS%20PCCSE%20Overview", "feature_navigation": true, "feature_search": true, "feature_download": true, "icon_url": "https://panoramaweb.org/labkey/logo.image?revision=281", "feature_access": true, "enrichment_analysis": false, "api": true, "feature_image_analysis": false, "processed_l1000_data": false, "name": "Panorama - PCCSE", "feature_visualization": true, "description": "Proteomics repository that provides access to the PCCSE P100 and GCP data", "order": 25}]